docs/pages/access-controls/access-graph/aws-sync.mdx

```diff
@@ -0,0 +1,219 @@
+Access Graph`.
+
+The setup process is intuitive, guiding you through the steps. You'll be
+prompted to create a new Teleport AWS integration if you haven't configured
+one already. Alternatively, you can opt for a previously established integration.
+
+Upon selecting or creating the integration, you'll be instructed to execute a
+bash script within your AWS Cloud Shell to configure the necessary permissions.
+
+<Details title="List of IAM Policies required for AWS Sync" closed>
+
+As part of the setup process, you'll be prompted to execute a bash script
+within your AWS Cloud Shell. This script's purpose is to generate an IAM
+Policy within your AWS account and link it to the AWS Role designated for
+your Teleport AWS Integration. The policy is designed with a set of read-only
+actions, enabling Teleport to access and retrieve information from resources
+within your AWS Account.
+
+The IAM policy includes the following directives:
+
+
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        // EC2 IAM
+        "ec2:DescribeInstances",
+        "ec2:DescribeImages",
+        "ec2:DescribeTags",
+        "ec2:DescribeSnapshots",
+        "ec2:DescribeKeyPairs",
+
+        // EKS IAM
+        "eks:ListClusters",
+        "eks:DescribeCluster",
+        "eks:ListAccessEntries",
+        "eks:ListAccessPolicies",
+        "eks:ListAssociatedAccessPolicies",
+
+        // RDS IAM
+        "rds:DescribeDBInstances",
+        "rds:DescribeDBClusters",
+        "rds:ListTagsForResource",
+        "rds:DescribeDBProxies",
+
+        // DynamoDB IAM
+        "dynamodb:ListTables",
+        "dynamodb:DescribeTable",
+        // Redshift IAM
+        "redshift:DescribeClusters",
+        "redshift:Describe*",
+        // S3 IAM
+        "s3:ListAllMyBuckets",
+        "s3:GetBucketPolicy",
+        "s3:ListBucket",
+        "s3:GetBucketLocation",
+        // IAM IAM
+        "iam:ListUsers",
+        "iam:GetUser",
+        "iam:ListRoles",
+        "iam:ListGroups",
+        "iam:ListPolicies",
+        "iam:ListGroupsForUser",
+        "iam:ListInstanceProfiles",
+        "iam:ListUserPolicies",
+        "iam:GetUserPolicy",
+        "iam:ListAttachedUserPolicies",
+        "iam:ListGroupPolicies",
+        "iam:GetGroupPolicy",
+        "iam:ListAttachedGroupPolicies",
+        "iam:GetPolicy",
+        "iam:GetPolicyVersion",
+        "iam:ListRolePolicies",
+        "iam:ListAttachedRolePolicies",
+        "iam:GetRolePolicy"
+      ],
+      "Resource": "*"
+    }
+  ]
+}
+
+```
+</Details>
+
+
+Once the IAM Policy has been successfully linked to the IAM role
+utilized by Teleport, you'll be prompted to specify the regions from
+which Teleport should import resources. This selection solely pertains
+to regional resources and does not impact global resources such as S3
+Buckets, IAM Policies, or IAM Users.
+
+If you're operating an on-premises cluster, you'll additionally need to
```

Comment:
```suggestion
If you're operating a self-hosted cluster, you'll additionally need to
```

Most people running Teleport in their own AWS environments would not consider that to be "on-premises" as it's an Amazon datacenter, not their own.